core/src/test/java/feast/core/it/BaseIT.java

```diff
@@ -0,0 +1,172 @@
+ * limitations under the License.
+ */
+package feast.core.it;
+
+import feast.core.config.FeastProperties;
+import feast.core.util.KafkaSerialization;
+import feast.proto.core.IngestionJobProto;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Table;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.hibernate.engine.spi.SessionImplementor;
+import org.junit.jupiter.api.*;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.config.KafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.listener.ConcurrentMessageListenerContainer;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@SpringBootTest
+@ActiveProfiles("it")
+@Testcontainers
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+public class BaseIT {
+
+  @Container public static PostgreSQLContainer<?> postgreSQLContainer = new PostgreSQLContainer<>();
+
+  @Container public static KafkaContainer kafka = new KafkaContainer();
+
+  @DynamicPropertySource
+  static void properties(DynamicPropertyRegistry registry) {
+
+    registry.add("spring.datasource.url", postgreSQLContainer::getJdbcUrl);
+    registry.add("spring.datasource.username", postgreSQLContainer::getUsername);
+    registry.add("spring.datasource.password", postgreSQLContainer::getPassword);
+    registry.add("spring.jpa.hibernate.ddl-auto", () -> "none");
+
+    registry.add("feast.stream.options.bootstrapServers", kafka::getBootstrapServers);
+  }
+
+  @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+  public class SequentialFlow {
+    @AfterAll
+    public void tearDown() throws SQLException {
```

Comment:
Is this `SequentialFlow` meant to be reused? If so, should the SQLException be removed or changed to Exception to make it more generalized?